webpack/config.js

```diff
@@ -40,13 +39,6 @@ const chunkFilename = IS_PRODUCTION ? "[chunkhash].chunk.js" : "[name].chunk.js"
 const devTool = IS_PRODUCTION ? "source-map" : "inline-cheap-source-map";
 
 var plugins = [
-  // Forked version of the RelayCompilerWebpackPlugin that supports
-  // relay-compiler 1.1.0
-  new RelayCompilerPlugin({
-    schema: path.join(__dirname, '../app/graph/schema.json'),
-    src: path.resolve(__dirname, '../app')
-  }),
```

Comment:
So I thought we avoided running relay-compiler manually by using this relay compiler webpack plugin, which webpack dev server and the build process run to create the required generated files. Is the trouble that these files aren't generated in time to be consumed by our own files?

Reply:
Also, does webpack 4 address any of this?

Reply:
I love javascript. 😭